kuma/static/js/wiki-compat-tables.js

```diff
@@ -0,0 +1,274 @@
+                });
+                betaSubmenuItems.push($betaError);
+
+                var $betaShowOld = $('<button />', { text: gettext('Show old table.'), 'class': 'button bc-old' });
+                $betaShowOld.on('click', function(){
+                    $('.bc-old').toggle();
+                    mdn.analytics.trackEvent({
+                        category: 'Compat Tables Show Old',
+                        action: location.pathname
+                    });
+                });
+                betaSubmenuItems.push($betaShowOld);
+
+                $(betaSubmenuItems).each(function(){
+                    var $newItem = $('<li />');
+                    $newItem.append(this);
+                    $newItem.appendTo($betaSubmenu);
+                });
+
+                $betaMenuWrapper.append($betaMenuTrigger).append($betaSubmenu);
+                $betaMenuWrapper.insertBefore($table);
+
+                $betaMenuWrapper.mozMenu().mozKeyboardNav();
+            }
+
+            // Function which closes any open history, opens the target history
+            // Acts as the "router" for open and close directives
+            function closeAndOpenHistory($td) {
+                if(animating) {
+                    return;
+                }
+
+                // If no cell is open at the moment, skip the "close" step and just open it
+                if(!$openCell) {
+                    return _open($td);
+                }
+
+                // If they clicks the same cell (are closing), we can leave now
+                var previousOpenCell = $openCell && $openCell.get(0);
+                if($td.get(0) === previousOpenCell) {
+                    $td = null;
+                }
+
+                // Close what's open, if anything, and open if $td has a value
+                hideHistory($td);
+            }
+
+            // Opens the history for a given item
+            function showHistory() {
+                if(animating) {
+                    return;
+                }
+
+                var $row = $openCell.closest('tr');
+                var $history = $openCell.find('.bc-history').outerWidth($table.width());
+
+                // get cell coords
+                var cellLeft = $openCell.offset().left;
+
+                // get cell left border
+                var cellLeftBorder = $openCell.css('border-left-width');
+
+                // get table coords
+                var tableLeft = $table.offset().left;
+
+                // left coord of table minus left coord of cell
+                var historyLeft = tableLeft - cellLeft - parseInt(cellLeftBorder, 10);
+
+                // get cell height
+                var cellTop = $openCell.outerHeight();
+
+                // get cell top border
+                var celltopBorder = $openCell.css('border-top-width');
+
+                // get cell bottom border
+                var cellBottomBorder = $openCell.css('border-bottom-width');
```

Comment:
You could combine this, with lines 187, and 175 as:

```
var cellBorders = $openCell.css(['border-top-width', 'border-left-width', 'border-bottom-width']);
```

This then gets all of the computed styles in one call, and returns an Object that you can use, for example:

```
var historyTop = cellTop - parseInt(cellBorders.border-bottom-width, 10) - parseInt(cellBorders.border-top-width, 10);
```

Reply:
I don't think I can use the dot notation with dashes in the names. I've changed it to `cellBorders['border-bottom-width']` and so-on.